addons/website/views/snippets/s_tabs.xml

```diff
@@ -64,32 +64,53 @@
         </div>
         <div data-js="NavTabsStyle" data-selector="section" data-target=".s_tabs_main">
             <we-select string="Style">
-                <we-button data-set-style="tabs" data-name="tabs_opt" data-trigger="horizontal_opt">Tabs</we-button>
-                <we-button data-set-style="pills" data-name="pills_opt" data-trigger="tabs_color_opt" data-trigger-value="">Buttons</we-button>
+                <we-button data-set-style="nav-underline" data-name="underline_opt">Underline</we-button>
+                <we-button data-set-style="nav-tabs" data-name="tabs_opt" data-trigger="horizontal_opt">Tabs</we-button>
+                <we-button data-set-style="nav-buttons" data-name="buttons_opt" data-trigger="horizontal_opt">Buttons</we-button>
+                <we-button data-set-style="nav-pills" data-name="pills_opt">Pills</we-button>
             </we-select>
-            <we-colorpicker string="Tabs Color" data-dependencies="tabs_opt" data-name="tabs_color_opt" data-select-style="true" data-css-property="background-color" data-color-prefix="bg-"/>
-            <we-select string="Direction" data-dependencies="pills_opt">
+            <we-colorpicker
+                string="Header Background"
```

Comment:
"Background" seems enough to me ;)

| this pr | suggestion |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/5b93f8dc-8037-4900-bcd9-23460772f6ad) | ![image](https://github.com/user-attachments/assets/c9df343f-2390-4ffb-9730-ed88d6004bf3) |